sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/parser/RealSyncCrypter.kt

```diff
@@ -131,6 +133,7 @@ class RealSyncCrypter(
             title = encrypt(bookmarkFolder.name, primaryKey),
             children = children,
             deleted = null,
+            clientLastModified = DatabaseDateFormatter.iso8601(),
```

Comment:
should this be the value in `bookmarkFolder`?

Reply:
yes. sorted